enumerable.rb

```diff
@@ -0,0 +1,131 @@
+module Enumerable
+  def my_each
+    return to_module unless block_given?
+
+    x = 0
+    while x < length
+      yield(to_a[x])
+      x += 1
+    end
+    self
+  end
+
```

Comment:
* [ ] should have working `my_each`  identical and returns the same thing as ruby's [`each`](https://ruby-doc.org/core-2.6.4/Array.html#method-i-each). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_each.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))

#### Errors
```bash
rspec ./spec/spec_enumerables_spec.rb:27 # enumerables my_each returns an Enumerator if no block is given
```